tests/fsharp/Compiler/CompilerAssert.fs

```diff
@@ -26,13 +38,13 @@ type ILVerifier (dllFilePath: string) =
     member this.VerifyILWithLineNumbers (qualifiedItemName: string, expectedIL: string) =
         ILChecker.checkILItemWithLineNumbers qualifiedItemName dllFilePath [ expectedIL ]
 
-[<RequireQualifiedAccess>]
-module CompilerAssert =
+[<AbstractClass;Sealed>]
+type CompilerAssert private () =
```

Comment:
Is this (and subsequence modifications to make functions static members) necessary?

Reply:
This, as I understand, is the proper way to make what a static class would be compiled as in C#. Honestly, I wish we would have a Static attribute and make it behave like a C# static class.

Reply:
No, I mean - why the change? It doesn't appear to be doing anything different than before.

Reply:
I want to use optional parameters and overloads for the functions in CompilerAssert. I need to make it a class, and I want that class to not be inheritable.

Reply:
But you're not using them? Or perhaps I'm missing something.

Note that we'll have to check that we don't collide the the new tests that have been ported over.

Reply:
I will be using them. Still WIP. Also, I will handle any conflicts from the rest of the ported tests.